src/prefect/client/client.py

```diff
@@ -253,6 +261,43 @@ def graphql(
         """
         Convenience function for running queries against the Prefect GraphQL API
 
+        Args:
+            - query (Any): A representation of a graphql query to be executed. It will be
+                parsed by prefect.utilities.graphql.parse_graphql().
+            - raise_on_error (bool): if True, a `ClientError` will be raised if the GraphQL
+                returns any `errors`.
+            - headers (dict): any additional headers that should be passed as part of the
+                request
+            - variables (dict): Variables to be filled into a query with the key being
+                equivalent to the variables that are accepted by the query
+            - token (str): an auth token. If not supplied, the `client.access_token` is used.
+            - retry_on_api_error (bool): whether the operation should be retried if the API returns
+                an API_ERROR code
+
+        Returns:
+            - dict: Data returned from the GraphQL query
+
+        Raises:
+            - ClientError if there are errors raised by the GraphQL mutation
+        """
+        if self._active_tenant_id is None:
+            self._init_tenant()
+        return self._graphql(
+            query, raise_on_error, headers, variables, token, retry_on_api_error
+        )
+
+    def _graphql(
+        self,
+        query: Any,
+        raise_on_error: bool = True,
+        headers: Dict[str, str] = None,
+        variables: Dict[str, JSONLike] = None,
+        token: str = None,
+        retry_on_api_error: bool = True,
+    ) -> GraphQLResult:
+        """
+        Convenience function for running queries against the Prefect GraphQL API
+
         Args:
```

Comment:
I'm not sure I would replicate all of the docstring here since it would always be the same as `self.graphql` and need to be maintained to match. I'm not sure how this works with our doc enforcement though.

Reply:
I replicated it for the doc enforcement ^^